src/Simulation/Simulators/QuantumSimulator/QubitManager.cs

```diff
@@ -64,12 +64,14 @@ protected override void ReleaseOneQubit(Qubit qubit, bool usedOnlyForBorrowing)
                 if (qubit != null)
                 {
                     bool areAllReleasedQubitsZero = ReleaseOne(this.SimulatorId, (uint)qubit.Id);
```

Comment:
why is this named `areAllReleasedQubitsZero` if it's only checking the status of the last qubit released? :)
it threw me off a little.

Reply:
Will rename appropriately